x-pack/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields.tsx

```diff
@@ -35,6 +35,10 @@ export interface HighlightedFieldsTableRow {
      * Highlighted field name (overrideField or if null, falls back to id)
      */
     field: string;
+    /**
+     * Highlighted overrideField name
```

Comment:
can you use the same description as you put in the `highlighted_fields_cell` file? To me it is a lot clearer

`Highlighted field's original name, when the field is overridden`